homeassistant/components/vicare/sensor.py

```diff
@@ -210,6 +210,62 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         unit_getter=lambda api: api.getGasConsumptionHeatingUnit(),
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_fuelcell_today",
```

Comment:
@borys-kupar what was the difference between this one and `gas_consumption_total_today` again? Does it really make sense to have both?

Reply:
Total gas consumption consists of 2 items for me: "fuel cell" and "peak load boiler". Those are optional things, that helps you understand better how gas is being consumed. See the screenshot from the ViCare app:
![Screenshot 2024-01-31 at 12 53 06](https://github.com/home-assistant/core/assets/1401727/6b8041c5-6beb-4802-b1d5-1222aa920752)

Reply:
And "fuel cell" and "peak load boiler" are two different heating systems?
I guess it would be nice to see that as well (maybe in another pr).

Reply:
it's probably dhw, I'll confirm and file a request if needed.